### ISSUE TITLE
can't use bootstrap.py to generate HPCCM container

### ISSUE THREAD

**Issue by chenxull** [NVIDIA/hpc-container-maker#157]
i use `sudo docker build -t hpccm -f Dockerfile .` to generate HPCCM container ,but meet some problem。
## ERROR
```
Sending build context to Docker daemon  6.144kB
Step 1/3 : FROM python:3-slim
 ---> ca7f9e245002
Step 2/3 : RUN pip install --no-cache-dir hpccm
 ---> Running in b34d42cd6c00
Collecting hpccm
  Downloading https://files.pythonhosted.org/packages/b8/bd/a88fe1e1fa0bf5b17f3a2d4d001050bdb1e2c89cd9bf5bb24c47b314b33b/hpccm-19.5.0.tar.gz (133kB)
    ERROR: Complete output from command python setup.py egg_info:
    ERROR: Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-biiwov0l/hpccm/setup.py", line 10, in <module>
        from hpccm.version import __version__
      File "/tmp/pip-install-biiwov0l/hpccm/hpccm/__init__.py", line 22, in <module>
        from hpccm.Stage import Stage
      File "/tmp/pip-install-biiwov0l/hpccm/hpccm/Stage.py", line 25, in <module>
        from hpccm.primitives.baseimage import baseimage
      File "/tmp/pip-install-biiwov0l/hpccm/hpccm/primitives/__init__.py", line 24, in <module>
        from hpccm.primitives.runscript import runscript
      File "/tmp/pip-install-biiwov0l/hpccm/hpccm/primitives/runscript.py", line 23, in <module>
        from six.moves import shlex_quote
    ModuleNotFoundError: No module named 'six'
    ----------------------------------------
ERROR: Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-biiwov0l/hpccm/
The command '/bin/sh -c pip install --no-cache-dir hpccm' returned a non-zero code: 1
```

**Comment by samcmill** [comment]
Hi @chenxull, thanks for the problem report.  I understand the issue and hope to have a fix ready soon.

**Comment by samcmill** [comment]
Fixed in version 19.5.1.
